server/core/src/miscServerFunct.cpp

```diff
@@ -2983,12 +2982,24 @@ irods::error get_script_output_single_line(
     const std::string&              script_name,
     const std::vector<std::string>& args,
     std::string&                    output ) {
-    bp::ipstream output_stream;
-    bp::child c(script_language, args, bp::std_out > output_stream);
-    output.clear();
-    std::getline(output_stream, output);
-    c.wait();
-    return SUCCESS();
+    namespace bp = boost::process;
+    try {
+        auto tmp = args;
+        tmp.insert(tmp.begin(), fmt::format("{}/scripts/{}", irods::get_irods_home_directory().string(), script_name));
+        bp::ipstream output_stream;
+        bp::child c(script_language, tmp, bp::std_out > output_stream);
+        output.clear();
+        if (!output_stream) {
+            return ERROR(FILE_READ_ERR, "Bad stream");
+        }
+        std::getline(output_stream, output);
+        c.wait();
+        return SUCCESS();
+    }
+    catch (bp::process_error& e) {
+        irods::experimental::log::server::error("Error starting process, {}", e.what());
+        return ERROR(SYS_FORK_ERROR, e.what());
```

Comment:
I think we need to use a different error code here.

Do we know what `e.what()` says in this case?

Reply:
You're probably right. Try passing it different inputs and see what `e.what()` contains.

Reply:
It is as it turns out mostly about not being able to find the program, so I'll handle it as such

Reply:
What did the messages look like?

Post one of the outputs you received.